ckanext/harvest/harvesters/ckanharvester.py

```diff
@@ -404,36 +409,30 @@ def import_stage(self, harvest_object):
                 # check if remote groups exist locally, otherwise remove
                 validated_groups = []
 
-                for group_name in package_dict['groups']:
+                for group_ in package_dict['groups']:
```

Comment:
This makes sense, I assume it was a remnant of the old API version 1/2 based harvester where `groups` is a list of strings.